packages/web-components/src/components/va-file-input/va-file-input.tsx

```diff
@@ -10,9 +10,9 @@ import {
   EventEmitter,
   State,
 } from '@stencil/core';
-import { i18next } from '../..';
+import i18next from 'i18next';
```

Comment:
this will probably result in this component not rendering translations if this component is installed independently in an application, e.g. outside of vets-website